src/rlgl.h

```diff
@@ -1797,6 +1798,45 @@ void rlglDraw(void)
 #endif
 }
 
+// Check and log OpenGL error codes
+void rlglCheckErrors() {
+#if defined(GRAPHICS_API_OPENGL_21) || defined(GRAPHICS_API_OPENGL_33) || defined(GRAPHICS_API_OPENGL_ES2)
+    int check = 1;
+    while (check) {
+        const GLenum err = glGetError();
+        switch (err) {
+            case GL_NO_ERROR:
+                check = 0;
+                break;
+            case 0x0500: // GL_INVALID_ENUM:
+                TRACELOG(LOG_WARNING, "GL_INVALID_ENUM");
+                break;
+            case 0x0501: //GL_INVALID_VALUE:
+                TRACELOG(LOG_WARNING, "GL_INVALID_VALUE");
```

Comment:
I try to categorize the TRACELOG() messages, maybe something like:
```c
TRACELOG(LOG_WARNING, "GL: Error detected: GL_INVALID_VALUE");
```

Reply:
I feel like this could be better with more detail, but I suppose it should be rare enough and probably relevant only to direct rlgl.h users, so a log notice with error code is enough.